client/config.go

```diff
@@ -19,6 +19,9 @@ type Config struct {
 	AuthToken string
 	// ChannelID is the id of the channel to start or join a call in.
 	ChannelID string
+	// ContextID is an id used to identify bot initiated sessions (e.g.
+	// recording/transcription)
+	ContextID string
```

Comment:
Just wondering if it can be named something more descriptive, to indicate what the ContextID means? I imagine I will forget what it means.

Reply:
For context (no pun intended :p), this is the same we are introducing in https://github.com/mattermost/mattermost-plugin-calls/pull/462 . The reason I kept it this abstract is that I could see more use cases in the future that don't necessarily map to what we are doing now. For the current purposes this could be easily called `JobID`, although it's a bit of an implementation details leak in my mind.

Reply:
If we do need to abstract it in the future, would it be easy to then abstract the field name?

Reply:
We'd need to either change the existing name (breaking change) or add a new field. Can be done, but could be costly.

Reply:
Personally I would prefer a more concrete name (since even now I've forgotten exactly what it holds), but your call. 👍